discovery-provider/default_config.ini

```diff
@@ -1,6 +1,6 @@
 [discprov]
 start_block = 0x0
-loglevel_flask = DEBUG
+loglevel_flask = INFO
```

Comment:
we could set this to debug in the discovery commonEnv.sh so local dev gets DEBUG logs